src/Compilers/CSharp/Portable/Symbols/TypeSymbol.cs

```diff
@@ -812,7 +812,7 @@ public ImmutableArray<SymbolDisplayPart> ToDisplayParts(CodeAnalysis.NullableFlo
 
         #region Interface member checks
 
-        protected SymbolAndDiagnostics FindImplementationForInterfaceMemberInNonInterfaceWithDiagnostics(Symbol interfaceMember)
+        protected SymbolAndDiagnostics FindImplementationForInterfaceMemberInNonInterfaceWithDiagnostics(Symbol interfaceMember, bool ignoreImplementationInInterfacesIfResultIsNotReady = false)
```

Comment:
>ignoreImplementationInInterfacesIfResultIsNotReady [](start = 134, length = 50)

Could you please add some documentation for the parameter `ignoreImplementationInInterfacesIfResultIsNotReady` hopefully explaining how it fits into the overall scheme for what and how interface implementations are found?